### DISCUSSION TITLE
ferPPLe 6.16 doker issue

### DISCUSSION THREAD

**Discussion by fabiobonfa** [frePPLe/frepple#409] [Q&A] [answered]
**I have connection problem with database, if I run:**

```
 sudo docker run \
  -e POSTGRES_HOST=172.17.0.8\
  -e POSTGRES_PORT=5432 \
  -e POSTGRES_USER=xxxxx \
  -e POSTGRES_PASSWORD=xxxxx \
  --name frepple-community-local \
  --publish 9091:80 \
  --detach \
  ghcr.io/frepple/frepple-community:latest
```

  
  **frepple try to connect to 127.0.0.1 instead of 172.17.0.8 ; here the log:**

```
  Waiting for the database to be ready,
Traceback (most recent call last):,
  File "/usr/bin/frepplectl", line 51, in <module>,
    execute_from_command_line(sys.argv),
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/__init__.py", line 381, in execute_from_command_line,
    utility.execute(),
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/__init__.py", line 375, in execute,
    self.fetch_command(subcommand).run_from_argv(self.argv),
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/base.py", line 323, in run_from_argv,
    self.execute(*args, **cmd_options),
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/base.py", line 364, in execute,
    output = self.handle(*args, **options),
  File "/usr/lib/python3/dist-packages/freppledb/execute/management/commands/createdatabase.py", line 98, in handle,
    with psycopg2.connect(**conn_params) as connection:,
  File "/usr/lib/python3/dist-packages/psycopg2/__init__.py", line 127, in connect,
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync),
psycopg2.OperationalError: could not connect to server: Connection refused,
	Is the server running on host "localhost" (127.0.0.1) and accepting,
	TCP/IP connections on port 49153?,
could not connect to server: Cannot assign requested address,
	Is the server running on host "localhost" (::1) and accepting,
	TCP/IP connections on port 49153?,
,
```

  
  
  **I didn't found this problem on 6.15, please verify.
  Thanks
  Fabio**

**Comment by fabiobonfa** [comment]
Same problem on 6.17
Have a nice day.
Fabio

**Comment by jdetaeye** [comment, marked as answer]
I found a problem indeed.  It's fixed now in the 6.17.1 container image.

**Reply by fabiobonfa** [reply]
I confirm you that now it works.
Thanks for your support!
Fabio

**Comment by insevoo** [comment]
Hi,I failed to run docker 6.17.1 on Centos platform( information as attached). It seems to be the problem of db connection. However, I followed the following instructions to install and configure database:sudo yum install -y postgresqlsudo systemctl start postgresql-13sudo systemctl enable postgresql-13sudo su - postgrespsql template1 -c "create user frepple with password 'frepple' createrole"psql template1 -c "create database frepple encoding 'utf-8' owner frepple"psql template1 -c "create database scenario1 encoding 'utf-8' owner frepple"psql template1 -c "create database scenario2 encoding 'utf-8' owner frepple"psql template1 -c "create database scenario3 encoding 'utf-8' owner frepple"exitPlease let me know how to make sure that database is correctly set up. Thanks. 
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/frePPLe/frepple/discussions/409#discussioncomment-1454597",
"url": "https://github.com/frePPLe/frepple/discussions/409#discussioncomment-1454597",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
